trailblazer/store/api.py

```diff
@@ -200,32 +156,15 @@ def update_run_status(self, analysis_id: int, ssh: bool = False) -> None:
         if analysis.workflow_manager == WorkflowManager.TOWER.value:
             self.update_tower_run_status(analysis_id=analysis_id)
         elif analysis.workflow_manager == WorkflowManager.SLURM.value:
-            self.update_analysis_from_slurm_run_status(analysis_id=analysis_id, ssh=ssh)
+            self.update_analysis_from_slurm_run_status(analysis_id=analysis_id, use_ssh=use_ssh)
 
-    def update_analysis_from_slurm_run_status(self, analysis_id: int, ssh: bool = False) -> None:
+    def update_analysis_from_slurm_run_status(
+        self, analysis_id: int, use_ssh: bool = False
+    ) -> None:
         """Query slurm for entries related to given analysis, and update the analysis in the database."""
-        analysis: Analysis = self.get_analysis_with_id(analysis_id=analysis_id)
+        analysis: Optional[Analysis] = self.get_analysis_with_id(analysis_id=analysis_id)
```

Comment:
would checking for 
```Python 
if not analysis:
```
help us define a more accurate error or is the try, except covering this?

Reply:
It should be handled by the try. Probably need more refactoring to raise more specific exceptions in called functions, but out of scope here.